h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -795,7 +795,7 @@ else if (s.equals("-jks_pass")) {
       else if (s.equals("-internal_secure_connections")) {
         internal_secure_connections = true;
       }
-      else if (s.equals("-internal_security")) {
+      else if (s.equals("-internal_security_conf")) {
```

Comment:
Just thinking more, this will break existing users - could we make the original option as deprecated, but still preserve its functionality?